packages/studio/src/components/Viewport/defaults.tsx

```diff
@@ -0,0 +1,124 @@
+export interface Viewport {
+  name: string;
+  styles: { height: number; width: number } | null;
+  type: "desktop" | "mobile" | "tablet" | "other";
+  css: string;
+}
+
+export interface ViewportStyles {
+  name: string;
+  height: number;
+  width: number;
+}
+
+export interface ViewportMap {
+  [key: string]: Viewport;
+}
+
+export const EXPANDED_VIEWPORTS: ViewportMap = {
+  resetviewport: {
+    name: "Reset Viewport",
+    styles: null,
+    type: "other",
+    css: "h-full w-full",
+  },
+  iphonese: {
+    name: "iPhone SE",
+    styles: {
+      height: 667,
+      width: 375,
+    },
+    type: "mobile",
+    css: "aspect-[375/667]",
+  },
+  iphone14: {
+    name: "iPhone 14",
+    styles: {
+      height: 844,
+      width: 390,
+    },
+    type: "mobile",
+    css: "aspect-[390/844]",
+  },
+  galaxyzflip5folded: {
+    name: "Galaxy Z Flip5 Folded",
+    styles: {
+      height: 748,
+      width: 720,
+    },
+    type: "mobile",
+    css: "aspect-[720/748]",
+  },
+  galaxyzflip5unfolded: {
+    name: "Galaxy Z Flip5 Unfolded",
+    styles: {
+      height: 1760,
+      width: 720,
+    },
+    type: "mobile",
+    css: "aspect-[720/1760]",
+  },
+  pixel7: {
+    name: "Pixel 7",
+    styles: {
+      height: 800,
+      width: 360,
+    },
+    type: "mobile",
+    css: "aspect-[360/800]",
+  },
+  pixelfoldfolded: {
+    name: "Pixel Fold Folded",
+    styles: {
+      height: 720,
+      width: 372,
+    },
+    type: "mobile",
+    css: "aspect-[372/720]",
+  },
+  pixelfoldunfolded: {
+    name: "Pixel Fold Unfolded",
+    styles: {
+      height: 720,
+      width: 600,
+    },
+    type: "mobile",
+    css: "aspect-[600/720]",
+  },
+};
+
+export const MINIMAL_VIEWPORTS: ViewportMap = {
```

Comment:
This value seems unused? I don't have a strong opinion for whether we use EXPANDED_VIEWPORTS or MINIMAL_VIEWPORTS

Reply:
Yup unused rn, left it so if we wanna switch it's easy. I can ask Tom

Reply:
![Screenshot 2023-08-16 at 4 18 21 PM](https://github.com/yext/studio/assets/106991746/792a00d2-929e-4a57-b2a2-a1b79cb8a2ae)
![Screenshot 2023-08-16 at 4 19 23 PM](https://github.com/yext/studio/assets/106991746/ba2fc6d3-fe41-40e3-8395-2f1629f9c482)

Reply:
Tom said expanded!